docs/Extra Guides/createplugin.md

```diff
@@ -0,0 +1,206 @@
+---
+sidebar_position: 21
+---
+
+# 🟨 Create a Voter Weight Plugin
+
+## What is a Voter Weight Plugin?
+
+Realms is designed to be _modular_, allowing for the addition of _plugins_
+that can modify the voter weight of members in a DAO.
+
+By default, the voter weight of a member is equal to the number of tokens they hold.
+
+However, if a DAO enables a plugin, the voter weight of a member can be modified.
+
+Example plugins are:
+- [Quadratic Voting](qv.md): The voter weight of a member is proportional to the square root of the number of tokens they hold.
+- [Voter Stake Registry](VSR/vsrsetup.md): A voter's weight can be increased by locking tokens.
+- [NFT](../DAO Guides/NFT DAO/nftdao.md): The voter weight of a member is proportional to the number and type of NFTs they hold.
```

Comment:
the link is broken 

<img width="954" alt="image" src="https://github.com/solana-labs/governance-docs/assets/5817016/32582d1c-ae2d-4373-b541-1584fb577d5f">